src/Persoane/Curier/CurierBiciclist.java

```diff
@@ -0,0 +1,90 @@
+package Persoane.Curier;
+
+import Comanda.Comanda;
+import Persoane.Client;
+import Restaurant.Restaurant;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Objects;
+import java.util.PriorityQueue;
+
+public class CurierBiciclist extends Curier{
```

Comment:
CurierBiciclist and CurierSofer have some common fields like CNP, salariu, nrComenzi, comenzi - you can declare them in the Curier abstract class and not redeclare them in these 2 classes